licensing.md

```diff
@@ -0,0 +1,27 @@
+# Licensing
+
+## Each repository should include a licence file
+
+This should be called `LICENCE` or `LICENCE.md`. "License" is the U.S. English spelling.
+
+GitHub.com will still show licence details for the British English spelling.
+
+## Use MIT
+
+At GDS we use the [MIT License](https://opensource.org/licenses/MIT).
```

Comment:
It might be worth loosening the language a little here - MIT is generally a safe default but there may be [edge] cases where another licence is more appropriate - e.g. when extending/incorporating existing code etc.  Could add something along the lines of "if unsure ask your tech lead/tech arch" etc.  What do you think?